admin_ui/backend/api/agents.py

```
@@ -158,6 +191,37 @@ def templates():
         return json.load(f)
 
+
+@router.post("/agents/starter-set")
+def create_starter_set(body: StarterSetIn):
+    provider = (body.provider or "").strip()
+    pipeline = (body.pipeline or "").strip() or None
+    if not provider and not pipeline:
+        try:
+            from api.config import _read_merged_config_dict
+            config = _read_merged_config_dict() or {}
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)